test/parallel/test-net-connect-local-error.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+var common = require('../common');
```

Comment:
I copied that from another test. Should I make a PR that changes this for every test where it isn't already the new way?